dev_guide/builds.adoc

```diff
@@ -659,6 +659,30 @@ code. If your application exists inside a sub-directory, you can override the
 default location (the root folder) using this field.
 ====
 
+[[max-duration]]
+
+== Build maximum duration
+When defining a `*BuildConfig*`, you can define its maximum duration, setting
+`*maxDuration*` field, it is specified in seconds and by default it is not set.
+Maximum duration is counted from Build's StartTime and it means how long a build
```

Comment:
it's not counted from StartTime, StartTime is set when the pod is marked to Running.  But this duration starts counting as soon as the pod is created.

We don't have a timestamp for that, since we have "time at which the build was created" and "time at which the pod started running" but we don't have "time at which the pod was created"

in theory "time at which the build was created" should be the same as "time at which the pod was created" but it doesn't have to be since we might spend some time waiting for an imagestream to resolve, after creating the build, before actually creating the pod.